src/components/ViewRoutingSlip/RoutingSlipInfo.vue

```diff
@@ -126,29 +109,47 @@
         </v-btn>
       </v-card-actions>
     </v-card>
+    <ModalDialog
+      ref="modalDialogRef"
+      :title="$t('NSFWarningTitle')"
+      :text="$t('NSFWarningText')"
+      dialog-class="notify-dialog"
+      max-width="679"
+      max-height="310"
+      icon="mdi-alert-circle-outline"
+      iconColor="error"
+    >
+      <template v-slot:actions>
+        <v-btn large color="error" @click="updateStatus()" data-test="dialog-ok-button" class="px-5 font-weight-bold btn-actions">Place status to NSF</v-btn>
```

Comment:
we can have the text, title and button text in a variable so that in future if any other status need warning modal, we can update in composable function :)

Reply:
Will update in next PR